puppet/zulip/files/nagios_plugins/zulip_postgresql/check_fts_update_log

```diff
@@ -28,8 +27,10 @@ def report(state: str, num: str) -> None:
     exit(states[state])
 
 
-conn = psycopg2.connect(database="zulip")
-cursor = conn.cursor()
+os.environ["DJANGO_SETTINGS_MODULE"] = "zproject.settings"
+from django.db import connection
+
+cursor = connection.cursor()
```

Comment:
I think we might need the same logic we have in `process_fts_updates` itself, since Zulip supports `psycopg2` not being installed on a Zulip database server.

Reply:
I think you mean "supports `django` not being installed on a Zulip database server"?  If `psycopg2` isn't installed on the database server, then we have no way to talk to the database.  We install it on database hosts:
https://github.com/zulip/zulip/blob/master/puppet/zulip/manifests/process_fts_updates.pp#L7

...which is included in `postgresql_base`.

Reply:
Yes, sorry for the confusion.

Reply:
I think basically we should drop this commit and replace it with a new commit at the end to make `process_fts_updates` itself have a `--check` method , and have this call that method, so that we don't duplicate a bunch of messy import code.